common/protob/messages-management.proto

```diff
@@ -58,6 +58,23 @@ message Features {
     optional bool unfinished_backup = 27;       // report unfinished backup (equals to Storage.unfinished_backup)
     optional bool no_backup = 28;               // report no backup (equals to Storage.no_backup)
     optional bool recovery_mode = 29;           // is recovery mode in progress
+    repeated Feature features = 30;             // list of supported features
+    enum Feature {
+        Feature_Bitcoin = 1;
+        Feature_Altcoins = 2;
+        Feature_Binance = 3;
+        Feature_Cardano = 4;
+        Feature_Crypto = 5;
```

Comment:
What's that?

Reply:
Stuff defined in messages-crypto.proto

Reply:
Resolved in b729bc6f65bbff6d5c64f63d0014acb063c5f5ab